docs/core/tools/global-json.md

```diff
@@ -1,32 +1,96 @@
 ---
-title: global.json reference
-description: See the schema for the global.json file, which permits setting the .NET Core tools version.
-author: blackdwarf
+title: global.json overview
+description: Learn how to use the global.json file to set the .NET Core SDK version when running .NET Core CLI commands.
+author: mairaw
 ms.author: mairaw
-ms.date: 04/05/2017
+ms.date: 06/28/2018
+ms.custom: "updateeachrelease"
 ---
-# global.json reference
+# global.json overview
 
-The *global.json* file allows selection of the .NET Core tools version being used through the `sdk` property.
+[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
 
-.NET Core CLI tools look for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+The *global.json* file allows you to define which .NET Core SDK version is used when you run .NET Core CLI commands.
+
+.NET Core SDK looks for a *global.json* file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+
+## global.json schema
+
+### sdk
 
-## sdk
 Type: Object
 
-Specifies information about the SDK.
+Specifies information about the .NET Core SDK to select.
+
```

Comment:
I'm seeing enough confusion that I would like to add the following, perhaps as a note:

Selecting the .NET Core SDK is independent from specifying the .NET Core Runtime your project targets. You specify the runtime [[ a few words and links for how you select target runtime ]]. The .NET Core SDK version indicates which versions of tools are used. In general you want to use the latest tools, and thus no **global.json** is needed.